spec/dummy/config/initializers/access_policies.rb

```diff
@@ -1 +1,3 @@
-OSU::AccessPolicy.register(OpenStax::Accounts::Account, AccountAccessPolicy)
+Rails.application.config.to_prepare do
+  OSU::AccessPolicy.register(OpenStax::Accounts::Account, AccountAccessPolicy)
```

Comment:
I think this class doesn't reload, so I'm not sure putting the register call in to_prepare is the right thing

Reply:
[context](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#autoloading-during-initialization)

Reply:
Hm yeah I see the link, but this class shouldn't be autoloaded...
Was it throwing the warning without `to_prepare`?